massa-execution/Cargo.toml

```diff
@@ -17,8 +17,10 @@ serde_json = "1.0"
 thiserror = "1.0"
 tokio = { version = "1.11", features = ["full"] }
 tracing = {version = "0.1", features = ["max_level_debug", "release_max_level_debug"] }
-assembly_simulator = { git = "https://github.com/massalabs/massa-sc-runtime", tag = "v0.2.8" }
+assembly_simulator = { git = "https://github.com/massalabs/massa-sc-runtime", branch = "extend_abis" }  # TODO tag = "v0.2.9"
```

Comment:
Prepare the PR with  the `tag = "v0.3.0"` even if it fails for the moment. The rest looks ok for me.

Reply:
I will, first I want to test the whole thing in Labnet, then I will merge the massa-runtime-sc side, tag it and then merge the massa side

Reply:
of course :) just we may don't want to merge it

Reply:
version bump as a followup here: https://github.com/massalabs/massa/pull/2167